src/js/first_parties/twitter.js

```diff
@@ -0,0 +1,16 @@
+function unwrapTwitterURLs() {
+  let aElems = document.querySelectorAll("a[data-expanded-url][href^='https://t.co/']");
+  let n = aElems.length;
+  for (let i = 0; i < n; i++) {
+    let elem = aElems[i];
+    let deu = elem.getAttribute("data-expanded-url");
+    if (deu && (deu.startsWith("https://") || deu.startsWith("http://"))) {
+      elem.href = deu;
+      elem.addEventListener("click", function (e) {
```

Comment:
I tested that it works, yeah. I put a break point on the click action and verified we no longer passed control to twitter. I was trying to unittest it, but was having trouble making the mocked event listeners get called in the right order.